src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityDialectIT.java

```diff
@@ -44,27 +49,72 @@ static void beforeAll() throws SQLException, BucketAccessException, InterruptedE
         createTenantsTestTable();
         createRolesTestTables();
         createRolesAndTenantsTestTable();
-        createVirtualSchema();
+        createConnection();
+        createAdapterScript();
+        createVirtualSchemaJdbc();
+        createVirtualSchemaJdbcWithLocal();
+        createVirtualSchemaExa();
+        createVirtualSchemaExaLocal();
         createUser("RLS_USR_1");
         createUser("RLS_USR_2");
         createUser("RLS_USR_3");
         createUser("RLS_USR_4");
     }
 
-    private static void createVirtualSchema() throws SQLException, InterruptedException {
+    private static void createConnection() throws SQLException {
         statement.execute("CREATE CONNECTION JDBC_EXASOL_CONNECTION " //
                 + "TO 'jdbc:exa:localhost:8888' " //
                 + "USER '" + container.getUsername() + "' " //
                 + "IDENTIFIED BY '" + container.getPassword() + "'");
+    }
+
+    private static void createAdapterScript() throws SQLException, InterruptedException {
         statement.execute("CREATE OR REPLACE JAVA ADAPTER SCRIPT RLS_SCHEMA.ADAPTER_SCRIPT_EXASOL_RLS AS " //
                 + "%scriptclass com.exasol.adapter.RequestDispatcher;\n" //
                 + "%jar /buckets/bfsdefault/default/" + ROW_LEVEL_SECURITY_JAR_NAME_AND_VERSION + ";\n" //
                 + "/");
         TimeUnit.SECONDS.sleep(20); // FIXME: need to be fixed in the container
-        statement.execute("CREATE VIRTUAL SCHEMA virtual_schema_rls USING RLS_SCHEMA.ADAPTER_SCRIPT_EXASOL_RLS " //
+    }
+
+    private static void createVirtualSchemaJdbc() throws SQLException {
```

Comment:
Reduce redundancy by adding a parameter that accepts a key-value map to the create script.

Reply:
done